test/examples/ref/cpw/lumped_adaptive/error-indicators.csv

```diff
@@ -1,2 +1,2 @@
                     Norm,                 Minimum,                 Maximum,                    Mean
-        +7.056476189e-01,        +1.271835950e-06,        +1.698826275e-02,        +8.754978767e-04
```

Comment:
Just confirming: the discrepancy in these indicators for the `lumped_adaptive` and `wave_adaptive` cases 
are from different samples making their way into the PROM, as the linear grid shouldn't have been affected by these changes I believe.

Reply:
Yes that's what I'd expect, different high-dimensional model samples due to the random sampling. This will be addressed by #121 (maybe worth moving that up in priority just for that feature itself).